src/test.cpp

```diff
@@ -171,4 +170,4 @@ void Test::waterfallGradient()
 
 }
 
-QTEST_MAIN(Test)
```

Comment:
Avoid style changes.

Reply:
Our guideline is a newline at the end of the file.

Reply:
removed the change